examples/Nano33SenseSend/Nano33SenseSend.ino

```diff
@@ -65,15 +66,17 @@ void loop()
 {
   if (NRF_I2S->EVENTS_TXPTRUPD != 0)
   {
-    // Transfer is complete, start it with the preprocessed data
-    current = (current + 1) % NUM_BUFFERS;
-    NRF_I2S->TXD.PTR = (uint32_t)(data[current]);
-    NRF_I2S->EVENTS_TXPTRUPD = 0;
-
-    // Process next buffer
-    next = (current + 1) % NUM_BUFFERS;
-    chirp_connect_error_code_t err = chirp_connect_process_shorts_output(chirp, data[next], BUFFER_SIZE);
+    if (current == 0) {
```

Comment:
@dinuchiriac we can remove this `current` variable and instead use 

`if (current_buffer == buffer_one)`

Reply:
In fact in can be reduced to a one liner with a ternary statement..

`current_buffer = current_buffer == buffer_one ? buffer_two : buffer_one;`